core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -110,9 +125,19 @@ public Builder addAllConfig(Map<String, String> properties) {
       return this;
     }
 
+    public Builder addCredential(Credential credential) {
+      credentials.add(credential);
+      return this;
+    }
+
+    public Builder addAllCredentials(List<Credential> credentialsToAdd) {
```

Comment:
Nit: `credentialsToAdd` -> `credentials`

Reply:
Ah probably checkstyle complains about the hidden field being called credentials, nvm all good.

Reply:
yeah it's due to checkstyle